src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultControllerModelBuilder.cs

```diff
@@ -58,6 +59,17 @@ public ControllerModel BuildControllerModel([NotNull] TypeInfo typeInfo)
                 }
             }
 
+            var controllerType = typeInfo.AsType();
+            foreach (var propertyInfo in controllerType.GetProperties(BindingFlags.Public | BindingFlags.Instance))
```

Comment:
Discussed and we will be consistent with ModelBinding in general ... 
so this stays and we ignore properties with private setter in `DefaultControllerActionArgumentBinder`